card/mod/standard/set/all/rich_html/modal.rb

```diff
@@ -115,11 +115,6 @@ def close_modal_window
   end
 
   def pop_out_modal_window
-    # we probably want to pass on a lot more params than just view,
-    # but not all of them
-    # (eg we don't want layout, id, controller...)
-    popout_params = params[:view] ? { view: params[:view] } : {}
-    link_to icon_tag(:new_window), path: popout_params,
-                                   class: "pop-out-modal close"
+    link_to icon_tag(:new_window), path: {}, class: "pop-out-modal close"
```

Comment:
params[:view] is almost always "bridge" in this context.  I think the most useful thing is just a clean link to the card.